### PR TITLE
Do not delete local artifacts that would have failed to sync to gcs

### PR DESCRIPTION
**Pull request by Ekleog-NEAR**
I’ve deployed this to worker1 by checking out the commit in /home/nayduck and restarting nayduck-fuzzer.service, and it seems to be working fine.

However, I’ve noticed that the workers appear to be on a nayduck version from ~last november, that does not include any of the recent changes… any idea how that happened? (so what I exactly did was cherry-pick this commit on top of what was running to not break other stuff)

### CODE REVIEW

File: fuzzers/main.py [hunk 1]
```diff
@@ -254,12 +255,12 @@ def _reset_to_gcs(self, path: pathlib.Path,
             f'Resetting path {self.dir}/{path} to GCS {self.version}/{path}/\n')
         log_file.flush()
         utils.mkdirs(self.dir / path)
+        cmd = ['gsutil', '-m', 'rsync']
+        if not is_artifacts:
+            cmd.append('-d')
```
Comment (andrei-near) [inline fuzzers/main.py:260]: great catch!

**Review by andrei-near — APPROVED** [review]

**Comment by Ekleog-NEAR** [comment]
Hmm but then… how come https://github.com/near/nayduck/pull/41 doesn’t seem to have been deployed? (`git status` in `/home/nayduck/nayduck` doesn’t show it)

That makes me kinda worry about deploying this PR, seeing how I’d be pulling in all these changes too at a time where nayduck finally seems to work 😅

**Comment by andrei-near** [comment]
Building of binaries only happens once, on the Nayduck builder instance, and orkers only scp the binaries from builder when running tests. Nayduck builder instance code update was deployed already

**Comment by Ekleog-NEAR** [comment]
Ooh ok I get it now, thank you! Then I’ll merge this, deploy to all workers and restart all the fuzzers systemd services :)

**Comment by Ekleog-NEAR** [comment]
Done; I’ve successfully run this command on all nayduck workers:
```bash
$ sudo -u nayduck sh -c "cd ~/nayduck && git pull --ff-only" && sudo systemctl restart nayduck-fuzzer
```

**Pull request merged**